README.md

```diff
@@ -1,16 +1,61 @@
 
-The coordinate types LV95 as well as the older LV03 are supported. More information about the swiss coordinate systems [here](https://opendata.swiss/de/dataset/bezugsrahmenwechsel-lv03-lv95-koordinatenanderung-lv03-lv95) and [here](https://de.wikipedia.org/wiki/Schweizer_Landeskoordinaten).
+In order to use the collected borehole data efficiently, it is critical that both metadata as well as geological information is digitally stored in a structured database. However, the relevant data for most boreholes that are received by swisstopo, is contained in PDF-files that lack a standardized structure. Older data is often only available in the form of a scanned image, obtained from a printed document or from a microfiche. Manually entering all the relevant data from these various sources is not feasible, given the large amount of boreholes and the continuous influx of new data.
 
-Only German and French borehole profiles are supported as of now.
+Therefore, the goal of this project is to automate the extraction of structured data from borehole profiles as much as possible. As far as swisstopo is concerned, the use case is to integrate the data extraction pipeline with the application boreholes.swissgeol.ch ([Github](https://github.com/swisstopo/swissgeol-boreholes-suite)), where a user interface for efficient quality control of the automatically extracted data will also be implemented.
+
+All code and documentation is published in this Github repository as open source software. All other persons, companies or agencies who manage borehole data of their own, are welcome to use the data extraction pipeline on their own data and to contribute to the project with their own improvements/additions.
+
+### Extracted properties
+
+Below is a list of the most relevant properties for the extraction of structure data from borehole profiles. Properties that can be automatically extracted by the current pipeline are in **bold**. Properties for which the implementation of automatic extraction is actively being worked on, are in _italics_.
+
+* Metadata
+  * **Coordinates**
+  * _Date_
+  * _Drilling method_
+* Lithology / stratigraphy
+  * **Depths** (upper and lower bound of each layer)
+  * **Material descriptions** (as plain text)
+  * USCS classification, color, consistency, plasticity...
```

Comment:
Why not italic or bold?

Reply:
Not being actively worked on (at least not planned for the rest of the year). Same for the others.